data/implementations.yaml

```diff
@@ -620,7 +620,18 @@
   tags:
     - swift
     - kitura
-    
+
+"Swift / Kitura / Redis":
+  description:
+    A <a href="http://www.swift.org">Swift</a> implementation using
+    <a href="https://github.com/IBM-Swift/Kitura">Kitura</a> for web.
+   sourcecode_url: https://github.com/IBM-Swift/todolist-redis
```

Comment:
the `sourcecode_url`, `live_url` and `tags` fields are all indented by one space too many.